### PR TITLE
FAT-3170:Fix failed test in master

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

**Review by TetianaParanich — APPROVED** [review]

File: cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js [hunk 1]
```diff
@@ -61,20 +61,21 @@ describe('ui-finance: Funds', () => {
...
 
   after(() => {
-    cy.loginAsAdmin({ path:SettingsMenu.acquisitionUnitsPath, waiter: AcquisitionUnits.waitLoading });
-    AcquisitionUnits.unAssignAdmin(defaultAcquisitionUnit.name);
-    AcquisitionUnits.delete(defaultAcquisitionUnit.name);
-    cy.visit(TopMenu.fundPath);
+    cy.loginAsAdmin({ path:TopMenu.fundPath, waiter: Funds.waitLoading });
     FinanceHelp.searchByName(defaultfund.name);
     FinanceHelp.selectFromResultsList();
     Funds.deleteFundViaActions();
```
Comment (penfriend) [inline cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js:67]: please explain why you need this wait

Comment (penfriend) [reply cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js:67]: line 68

Comment (IhorBohdan) [reply cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js:67]: Robot sometimes manages to delete everything, but it happens that the test falls as not enough time to clean the data of fund.on this I decided to put cy.wait(), so he just had time to work out.

Comment (penfriend) [reply cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js:67]: write a comment wait until the message dissappeared, wait until object/element dissapear. you can use in this case the cy.expect to something not exist

File: cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js [hunk 2]
```diff
@@ -61,20 +61,21 @@ describe('ui-finance: Funds', () => {
...
     FiscalYears.deleteFiscalYearViaApi(defaultFiscalYear.id);
+    cy.visit(SettingsMenu.acquisitionUnitsPath);
+    AcquisitionUnits.unAssignAdmin(defaultAcquisitionUnit.name);
+    AcquisitionUnits.delete(defaultAcquisitionUnit.name);
+
     Users.deleteViaApi(user.userId);
   });
 
-  // Test is failed. FAT-3199
   it('C163928 Test acquisition unit restrictions for Fund records (thunderjet)', { tags: [testType.criticalPath, devTeams.thunderjet] }, () => {
```
Comment (penfriend) [inline cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js:79]: line 83  // Need to wait,while dato is load
dato?

**Review by penfriend — APPROVED** [review]

**Pull request merged**